backend/app/controllers/spree/admin/store_credits_controller.rb

```diff
@@ -0,0 +1,94 @@
+module Spree
+  module Admin
+    class StoreCreditError < StandardError; end
+
+    class StoreCreditsController < Spree::Admin::BaseController
+
+      before_filter :load_user
+      before_filter :load_categories, only: [:new, :edit]
+      before_filter :load_store_credit, only: [:new, :edit, :update]
+      before_filter :ensure_unused_store_credit, only: [:update]
+
+      def index
+        @store_credits = @user.store_credits.reverse_order
+      end
+
+      def create
+        @store_credit = @user.store_credits.build(
+          permitted_store_credit_params.merge({
+            created_by: try_spree_current_user,
+            action_originator: try_spree_current_user,
+          })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.